common/src/main/java/org/jboss/pnc/bacon/common/Path.java

```diff
@@ -0,0 +1,8 @@
+package org.jboss.pnc.bacon.common;
+
+public class Path {
```

Comment:
Conflicting with java.nio.file.Path, for other use I would prefer name to be changed as we discussed, for this change I checked code and there already is Config class containing config location. You can improve our code and unify usages and setup of config file.